tpu-provisioner/admission_controller/manifests/manifest.yaml

```diff
@@ -75,4 +71,24 @@ spec:
       volumes:
       - name: certs-volume
         secret:
-          secretName: admission-tls
\ No newline at end of file
+          secretName: admission-controller-tls
+---
+apiVersion: cert-manager.io/v1
```

Comment:
Will any more changes be needed to complete cert-manager integration/support?

Reply:
This should work out of the box, the cert details get injected into the mutating webhook.